ci/documentation/publish.sh

```diff
@@ -28,16 +28,17 @@ if [[ -z "${GITHUB_GIT_USER_EMAIL+x}" ]]; then
     exit 1;
 fi
 
-DOCS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-source "$DOCS_DIR"/../ci/_environment.sh
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+# shellcheck source=ci/_environment.sh
+source "$SCRIPT_DIR"/../_environment.sh
 
 docker run --rm \
-        --volume "$DOCS_DIR/..":/app \
+        --volume "$SCRIPT_DIR/../..":/app \
         --volume "${SSH_AUTH_SOCK:-/dev/null}":/tmp/ssh_auth_sock \
         --env "SSH_AUTH_SOCK=/tmp/ssh_auth_sock" \
         --env "LOCAL_USER_ID=$(id -u)" \
         --env "GITHUB_GIT_USER_NAME=${GITHUB_GIT_USER_NAME}" \
         --env "GITHUB_GIT_USER_EMAIL=${GITHUB_GIT_USER_EMAIL}" \
         -w="/app" \
-        ${DOCUMENTATION_IMAGE} \
-        sh -c "docs/_publish_internal.sh"
+        "${DOCUMENTATION_IMAGE}" \
+        bash -c "ci/documentation/_publish_internal.sh"
```

Comment:
Moved scripts for documentation to `ci/documentation` directory to separate the concerns.